starknet-foundry/crates/cast/src/starknet_commands/multicall.rs

```diff
@@ -105,3 +124,24 @@ pub async fn multicall(
 
     Ok(())
 }
+
+pub fn new(maybe_output_path: Option<Utf8PathBuf>, overwrite: bool) -> Result<()> {
+    if let Some(output_path) = maybe_output_path {
+        if output_path.exists() {
+            if !output_path.is_file() {
+                bail!("output file cannot be a directory");
+            }
+            if !overwrite {
+                bail!(
+                    "output file already exists, if you want to overwrite it, use the `overwrite` flag"
+                );
+            }
+        }
+        std::fs::write(output_path.clone(), DEFAULT_MULTICALL_CONTENTS)?;
+        println!("Multicall template successfully saved in {output_path}");
+    } else {
+        println!("{DEFAULT_MULTICALL_CONTENTS}");
```

Comment:
This is fine, but We are using a convention for all the other commands, that they do not print anything (they have `print_*` functions that are invoked in main. Perhaps we should stick to this convention? @war-in

Reply:
I understand you want to stick to the convention. The problem is, there's no additional logic for the print function here. So this will look like this:
```
pub fn print_new_result(new_result: String) -> Result<()> {
    println!("{new_result}");
    Ok(())
}
```
It doesn't bring anything new, just calls `println` and that's it.
I'm going to add this but I think this is redundant.

Reply:
[add print fn](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/8300df3309f03fc8cf2f12bfd04f6b2681a17c77)